src/App.tsx

```diff
@@ -86,6 +87,14 @@ export const App = ({ shortUrl, user }: Props) => {
     });
   }, [filters, shortUrl]);
 
+  useEffect(() => {
+    setLoading(true);
```

Comment:
technically this will cause a race condition with: https://github.com/guardian/discussion-rendering/pull/58/files#diff-46ef2374da6211486f3a408db68b2bcbR82

but this is beyond the scope of this PR